### PR TITLE
[Chore] Stories Page Improvement

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description

- [x] Use Tabs & Shallow routing

Fixes # (issue)

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/pages/stories/[[...slug]].js [hunk 1]
```diff
@@ -37,15 +33,10 @@ export default function Index({
       {archive ? (
         <>
           <Hero {...blocks?.otherHero} />
```
Comment (kilemensi) [inline src/pages/stories/[[...slug]].js:35]: Shouldn't Hero now move into `StoriesPage`?

File: src/components/StoriesPage/index.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    [breakpoints.up("md")]: {
+      padding: `${typography.pxToRem(60)} 0 ${typography.pxToRem(80)}  `,
+    },
+  },
+  section: {},
+}));
+
+function StoriesPage({
+  activeCategory,
+  categoriesPosts,
```
Comment (kilemensi) [inline src/components/StoriesPage/index.js:22]: Why not just `items`?

**Review by kilemensi — APPROVED** [review]
🚀 

----

Much better that page reloading.

File: src/components/Card/useStyles.js [hunk 3]
```diff
@@ -10,7 +10,10 @@ const useStyles = makeStyles(() => ({
     backgroundColor: "#FFFFFF",
     boxShadow: "0px 3px 10px #0000001A",
   },
-  focusHighlight: {},
+  focusHighlight: {
+    background: "transparent",
+  },
+  cardActionRoot: {},
```
[the remark this answers is not in the document]
Comment (KhadijaMahanga) [reply src/components/Card/useStyles.js:16]: @kilemensi yes, it doesn't. I will add `focusVisible` too. Card Action Area is highlighting the element by applying separate styles with the focusVisibleClassName.

**Pull request merged**